### DISCUSSION TITLE
Icons on files with specific extensions

### DISCUSSION THREAD

**Discussion by Angelk90** [community/community#139026] [Repositories]
### Select Topic Area

General

### Body

It would be nice to have for each extension its own icon like ide do for example vscode.
Instead of seeing a simple file icon.

| Github                                                                | Vscode                                                                 |
|----------------------------------------------------------------------------|----------------------------------------------------------------------------|
| ![Screenshot 1](https://github.com/user-attachments/assets/81d5030b-af38-4572-a4d7-7d96889db348) | ![Screenshot 2](https://github.com/user-attachments/assets/e340682d-b051-485d-84c4-94dc947bdb1f) |

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐
